src/components/AddBook.js

```diff
@@ -13,6 +14,7 @@ function AddBook() {
         <input id="author" type="text" placeholder="Author" />
         <input id="category" type="text" placeholder="Category" />
         <button
```

Comment:
- [ ] It would be nicer to add gaps between form elements so that your design matches the Zeplin design.
- Your design:
<img width="731" alt="image" src="https://user-images.githubusercontent.com/77190203/174388533-6c01bca1-b573-4b8f-ba49-2c8a40f38b0a.png">


- Zeplin design:
<img width="511" alt="image" src="https://user-images.githubusercontent.com/77190203/174388473-c402ebee-cacb-4667-bcbc-0b3c7522eb10.png">